test/auth/auth.spec.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-undef */
+let chai = require('chai')
+
+let chaiHttp = require('chai-http')
+
+// eslint-disable-next-line no-unused-vars
+let server = require('../../src/server.js')
+
+const { editUser } = require('../../src/controllers/user/handlers')
+
+let should = chai.should()
+
+chai.use(chaiHttp)
+
+var data = require('./shared.js')
+
+const application = require('../../src/app')
+
+const { PORT } = process.env
+let app = 'http://localhost:' + PORT
+
+let basePath = '/v1/auth'
+
+describe('Testing auth', () => {
+  describe('test auth/register-local', () => {
+    /**
+     * Test creating new user
+     */
+    describe('Test creating new user (customer) ', () => {
+      it('Creating new user (customer) ', (done) => {
+        chai
+          .request(app)
+          .post(basePath + '/register-local')
+          .send(data.customer)
+          .end((err, res) => {
+            should.not.exist(err)
+            res.should.have.status(201)
+            res.body.should.be.a('object')
+            res.body.email.should.equal(data.customer.email)
```

Comment:
If you are testing the equality between data.customer.email and the body.email it's mean that body it's an object, so the first is useless